cs/filesystem.texy

```diff
@@ -0,0 +1,97 @@
+
+Práce se souborovým systémem - Nette\Utils\FileSystem
+*****************************************************
+
+.[perex]
+[Nette\Utils\FileSystem |api:] je statická třída s užitečnými funkcemi pro práci se souborovým systémem.
+
+
+Všechny příklady předpokládají vytvořený alias:
+
+/--php
+use Nette\Utils\FileSystem;
```

Comment:
Mě to přišlo užitečné. Díval jsem se tam na namespace spíše než do perexu (asi proto, že je to formátované jako kód). 
Ale je pravda, že je to vlastně duplikovaná informace.

Teď bych to tam, ale nechal. To je na nové issue :-)